lib/v2/sdu/data.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+    wh: {
+        news: {
+            name: '山东大学（威海）新闻网',
+            route: '/news',
+            source: ['/'],
+            titlePrefix: '(威海)新闻网|',
+            docs: 'https://docs.rsshub.app/university.html#shan-dong-da-xue-wei-hai-xin-wen-wang',
```

Comment:
```suggestion
            docs: 'https://docs.rsshub.app/university.html#shan-dong-da-xue-wei-hai',
```
This should point to H2 heading